features/test-1.feature

```diff
@@ -1,8 +1,38 @@
-Feature: Richdeps/Behave test
- TestA requires (TestB or TestC), TestA recommends TestC
+Feature: Richdeps/Behave test (install remove test)
+
+ Version of packages: 1.0.0-1
+ TestA	Requires	TestB
+ TestD	Requires	TestE = 1.0.0-1
+ TestF	Requires	TestG >= 1.0.0-1, TestH = 1.0.0-1
+ TestI	Requires	TestJ >= 1.0.0-2 (cannot be installed)
+ TestK	Requires	TestM
+ TestL	Requires	TestM
 
 Scenario: Install TestA from repository "test-1"
  Given I use the repository "test-1"
  When I "install" a package "TestA" with "dnf"
- Then package "TestA, TestC" should be "installed"
- And package "TestB" should be "absent"
+ Then package "TestA, TestB" should be "installed"
+ And package "TestC" should be "absent"
+ When I "install" a package "TestA" with "dnf"
```

Comment:
Are the new tests running in new environment when it's called multiple times in the row?